required/tools/multicol-2019-10-01.sty

```diff
@@ -103,12 +103,6 @@
 %%
 \NeedsTeXFormat{LaTeX2e}[2018-04-01]
```

Comment:
Same here

Reply:
here it should go otherwise one gets a warning between 2017 and 2018 rollback